src/chrome/content/rules/Reuters.com.xml

```diff
@@ -59,6 +59,8 @@
 -->
 <ruleset name="Reuters.com (partial)">
 
+	<target host="reuters.com" />
+	<target host="www.reuters.com" />
```

Comment:
Please also update the comments on top.
See also:
af.reuters.com
lta.reuters.com
ara.reuters.com
ar.reuters.com
br.reuters.com
ca.reuters.com
cn.reuters.com
de.reuters.com
es.reuters.com
fr.reuters.com
in.reuters.com
it.reuters.com
jp.reuters.com
mx.reuters.com
ru.reuters.com
uk.reuters.com

widerimage.reuters.com
agency.reuters.com

Mismatch:
live.reuters.com
mobile.reuters.com

Refused:
sales.reuters.com